andela/ah-kifaru-backend
# #166840910 Setup unit testing with mocha and chai

#### What does this PR do?
Setup unit testing with mocha and chai

#### Description of Task to be completed?
- ensure all implemented functionality is tested henceforth
- installed mocha, chai, chai-http, @babel/register, @babel/polyfill as devDependencies

#### How should this be manually tested?
The following steps can be used:
1) Clone the repository 
2) Cd into the project folder 
3) Run `npm install` to install all project dependencies. 
4) On the terminal run `npm test`
5) There should be a response showing that 2 tests are passing

#### Any background context you want to provide?
- Ensure that node and npm are installed on your machine

#### What are the relevant pivotal tracker stories?
Chore 166840910: https://www.pivotaltracker.com/story/show/166840910

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/31288326/60205312-cb74af80-9848-11e9-88dc-faf105164b0e.png)
